.gitignore

```diff
@@ -1 +1,5 @@
-node_modules
\ No newline at end of file
+node_modules
+index.html
+*.css
+*.webp
+*.svg
```

Comment:
I wonder why we are missing trailing \n, do we need to improve the formatter?

Reply:
I don't really understand why it does skip this file? https://github.com/linz/style-js/blob/86905c5255343747594510fe8b6b56d8a63faf0e/.prettierrc.cjs#L8C3-L8C18 : this seems to work for other file, or I am not looking at the right place?